definition_of_ready.md

```diff
@@ -0,0 +1,34 @@
+# Definition of ready (DoR)
+- Each user story should fulfill the INVEST criteria (see e.g. [Wikipedia](https://en.wikipedia.org/wiki/INVEST_(mnemonic)), [scrum-in-der-praxis.de](https://scrum-in-der-praxis.de/glossary/invest))
+  - See https://www.knowledgehut.com/tutorials/scrum-tutorial/definition-of-ready
+    - Independent (of all others)
+    - Negotiable (not a contract for a specific feature)
+    - Valuable (a vertical slice)
+    - Estimable (to a decent approximation)
+    - Small (enough to fit in a single iteration)
+    - Testable (even if the test doesn't exist yet)
+
+## Further requirements
+- The user story is Well-defined:
+  - The user story should be written in the user story format, see for example [Wikipedia](https://en.wikipedia.org/wiki/User_story#Principle).
+  - e.g. `As a <role> I can <capability>, so that <receive benefit>`
+- Acceptance criteria are defined (see point 7 from [dzone.com](https://dzone.com/articles/10-tips-for-writing-good-user-stories))
+  - The conditions that have to be fulfilled such that the story is done are described. The description should allow any member of the dev team to decide whether the story is done.
```

Comment:
```suggestion
  - The description states clearly which conditions have to be fulfilled such that the story is done.
```

Reply:
What do you think? I omitted the "team"-part here, because I think the whole story should be easy to understand, which I included in the succestion below.